### PR TITLE
added DatabaseSystems in free-course-hi.nd and two courses in it

### PR DESCRIPTION
**Pull request by swastikgithub09**
## What does this PR do?
Add resource(s) 

### Description
one heading in index of "Database Systems" was added and two courses under it were inserted
### Why is this valuable (or not)?
one of major pillars of computer science , so students will need this very much
### How do we know it's really free?
free on youtube
### For book lists, is it a book? For course lists, is it a course? etc.
it is a course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * [C](#c)
 * [C++](#cpp)
 * [Data Structures](#data-structures)
+* [Database Systems](#database-systems)
```
Comment (eshellman) [inline courses/free-courses-hi.md:10]: Use 'database' or 'databases' to align with other lists

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -101,6 +102,12 @@
 * [Stack Playlist \| Interview Questions \| Coding \| Tutorials \| Data Structures](https://www.youtube.com/playlist?list=PL_z_8CaSLPWdeOezg68SKkeLN4-T_jNHd) - Aditya Verma
 
 
+### Databases
+
+* [DBMS(Database Management System)Complete Playlist](https://www.youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers (YouTube)
+* [DBMS(Database Management System)Placement Series](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp (YouTube)
```
Comment (davorpa) [inline courses/free-courses-hi.md:108]: - Don't invent titles! Copy as it is
- YT playlist don't need notes
- Complete course instructors
 
```suggestion
* [DBMS (Database Management System)Complete Playlist](https://www.youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers, `tch.:` Varun Singla
* [DBMS Placements Series 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp, `tch.:` Love Babbar, `tch.:` Lakshay Kumar
```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: courses/free-courses-hi.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-hi.md]: Solved at 207d267d2397e99062e811126c72978b7c10016f

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -101,6 +102,12 @@
 * [Stack Playlist \| Interview Questions \| Coding \| Tutorials \| Data Structures](https://www.youtube.com/playlist?list=PL_z_8CaSLPWdeOezg68SKkeLN4-T_jNHd) - Aditya Verma
 
 
+### Databases
+
+* [DBMS (Database Management System)Complete Playlist](https://www.youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers, `tch.:` Varun Singla
+* [DBMS Placements Series 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp, `tch.:` Love Babbar, `tch.:` Lakshay Kumar
```
Comment (eshellman) [inline courses/free-courses-hi.md:108]:
```suggestion
* [DBMS Placements Series 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp by Babbar
```

**Review by eshellman — CHANGES_REQUESTED** [review]
sorry for the back and forth

**Pull request merged**